app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+        self.table_name = "actors"
+
+    def create(self, first_name_: str, last_name_: str) -> None:

```

Comment:
I don't see the reason to add `_` to those variables. It's usually done to avoid shadowing names of built-in variables. Do you have any reason?